fastflags.go

```diff
@@ -18,17 +18,18 @@ type flagMap map[string]*flagHolder
 
 // newFastFlags returns a new, empty fastFlags instance.
 func newFastFlags() *fastFlags {
-	ff := new(fastFlags)
-	ff.flags.Store(make(flagMap))
-	return ff
+	return new(fastFlags)
 }
 
 func (ff *fastFlags) load() flagMap {
-	return ff.flags.Load().(flagMap)
+	if flags := ff.flags.Load(); flags != nil {
+		return *flags
+	}
+	return nil
 }
 
 func (ff *fastFlags) Get(key string) (*flagHolder, bool) {
-	if f, ok := ff.flags.Load().(flagMap)[key]; ok && f != nil {
+	if f, ok := ff.load()[key]; ok && f != nil {
```

Comment:
I explicitly tested this in the playground: with the 2-result `s, ok` syntax, its not a panic to index into a nil map (`ok` will just be `false`)

Reply:
also tested in `TestDefaultFastFlags`